spec.html

```diff
@@ -1278,8 +1278,8 @@ <h1>Property Attributes</h1>
             </tbody>
           </table>
         </emu-table>
-        <p>If the initial values of a property's attributes are not explicitly specified by this specification, the default value defined in <emu-xref href="#table-4"></emu-xref> is used.</p>
-        <emu-table id="table-4" caption="Default Attribute Values">
+        <p>If the initial values of a property's attributes are not explicitly specified by this specification, the default value defined in <emu-xref href="#table-default-attribute-values"></emu-xref> is used.</p>
+        <emu-table id="table-default-attribute-values" caption="Default Attribute Values">
```

Comment:
So we can link here "default value(s)" references in `ValidateAndApplyPropertyDescriptor` (and vice versa).

Reply:
```suggestion
 <emu-table id="table-default-attribute-values" caption="Default Attribute Values" oldids="table-4">
```
since there might be external links to `table-4`.

Reply:
Perfect, thank you. b575b33